app/models/thredded/messageboard.rb

```diff
@@ -33,8 +33,13 @@ class Messageboard < ActiveRecord::Base
              through:    :recently_active_user_details,
              source:     :user
 
+    belongs_to :thredded_messageboard_group
```

Comment:
Since we are in the `Thredded` module, the association name should just be `messageboard_group`.
Also it's always good to add `inverse_of` so that certain calls don't perform extra queries: `inverse_of: :messageboards`.